README.md

```diff
@@ -0,0 +1,101 @@
+# Auctionable Change API [![Build Status](https://travis-ci.com/Auctionable-Change/auctionable_change_api.svg?branch=master)](https://travis-ci.com/Auctionable-Change/auctionable_change_api)
+
+## Description
+
+**Auctionable Change API** is the backend API for our [Turing School of Software and Design](https://turing.io/) capstone project, **Auctionable Change**.
+
+The idea behind **A.change** is creating a platform in which an individual can post an item for sale/auction, set a minimum asking price and then select a charity to which the funds from that item's sale would be donated.
+
```

Comment:
I like this description! Nice and concise. Easy to understand. Maybe if we have time further on we can elaborate on the buyer side too?

Reply:
Yeah -- with the uploading receipt routine.  The FE might get that covered with their project description.